addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/hw/HwSerialBridgeHandler.java

```diff
@@ -64,9 +67,9 @@ public HwSerialBridgeHandler(Bridge bridge) {
     public void initialize() {
         logger.debug("Initializing the Lutron HomeWorks RS232 bridge handler");
         HwSerialBridgeConfig configuration = getConfigAs(HwSerialBridgeConfig.class);
-        serialPortName = configuration.serialPort;
-        baudRate = configuration.baudRate.intValue();
-        updateTime = configuration.updateTime;
+        serialPortName = configuration.getSerialPort();
+        baudRate = configuration.getBaudRate().intValue();
```

Comment:
Good catch.  I appreciate your careful review.  I'll fix these and address your other comments shortly.